### DISCUSSION TITLE
Can't disable laptop display in clamshell mode.

### DISCUSSION THREAD

**Discussion by luispinto23** [Manjaro-Sway/manjaro-sway#697] [Q&A] [answered]
So, first of all, thank you all for this amazing sway edition.
I am using manjaro-sway since yesterday on my laptop and I previously used the Sway community edition of EndeavourOS, so I was trying to connect my laptop to my external monitor and close the laptop lid to continue using only the external monitor, but I found that my laptop monitor is not turning off, as it did back in endeavouros, so I went checking their config and saw that they use[ a script](https://github.com/EndeavourOS-Community-Editions/sway/blob/main/.config/sway/scripts/clamshell.sh) to disable the laptop screen when the lid is closed.

I tried to use the same script but I am not having much success. I've also ran the `swaymsg output eDP-1 disable ` command to see if it works, but what happens is that the laptop screen is disabled and then enabled right away. I have made no changes in the configs of the distro, since it's a fresh install, can someone help me out here?

**Comment by boredland** [comment, marked as answer]
way-displays does maybe interfere, but they claim to offer such functionality too: https://github.com/alex-courtis/way-displays

**Reply by luispinto23** [reply]
Thank you, having a look at the repo and adding my user to the `input` group, as suggested there, seems to have solved the problem.

**Reply by boredland** [reply]
I guess we should find out why our user isn't in the input group by default (mine is, for some reason).
